blazor/combobox/form-validation.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Form Validation in Blazor ComboBox Component | Syncfusion
+description: Checkout and learn about Form validation with Blazor ComboBox component in Blazor Sever App and Blazor WebAssembly App.
+platform: Blazor
+control: ComboBox
+documentation: ug
+---
+
+# Form Validation in ComboBox
+
+## ComboBox inside edit form
+
+The `EditForm` validates all data annotation rules using the `DataAnnotationsValidator`. If the input is valid, the form will be ready to be submitted. If the input is invalid, an error message will be displayed until a valid value is chosen.
+
+In this following example, the `EditForm` component is used to wrap the ComboBox and the submit button. The `DataAnnotationsValidator` component is used to enable data annotation-based validation, and the `ValidationMessage` component is used to display the validation error message. The `Required` attribute is applied to the Name field to make it a required field.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/form-validation/combobox-inside-editform.razor %}
+
+{% endhighlight %}
+
+![Blazor ComboBox inside editform](./images/form-validation/blazor_combobox_inside-editform.png)
+
+## Form validation using injectable datasource
+
+The ComboBox component inside the Edit form with the required field validation and with the injectable datasource is mentioned in the following sample.

```

Comment:
No need to repeat the same content to explain about new datasource, if it is the case, move the content to databinding section.